tests/ascii_depth_profile_rheology.prm

```diff
@@ -0,0 +1,110 @@
+# This is a relatively realistic incompressible model setup.
```

Comment:
I am not sure you wanted to add this test? At the moment it does not use your new rheology plugin. But now that you use the new plugin inside the depth_dependent material model you also do not need an extra test, because it is already used in the `depth_dependent_box_file_simple.prm` test.